catalog/transformer/catalog-transformer-geojson-input/src/main/java/ddf/catalog/transformer/input/geojson/GeoJsonInputTransformer.java

```diff
@@ -95,14 +98,14 @@ public Metacard transform(InputStream input, String id)
         }
 
         final String propertyTypeName = (String) properties.get(METACARD_TYPE_PROPERTY_KEY);
-        MetacardImpl metacard = null;
+        final MetacardType metacardType;
 
         if (propertyTypeName == null || propertyTypeName.isEmpty() || metacardTypes == null) {
             LOGGER.debug(
                     "MetacardType specified in input is null or empty.  Assuming default MetacardType");
-            metacard = new MetacardImpl();
+            metacardType = attributeInjector.injectAttributes(BasicTypes.BASIC_METACARD);
```

Comment:
This should be done by CatalogFramework#transform, not by every transformer implementation.